src/open_inwoner/accounts/tests/test_action_views.py

```diff
@@ -354,17 +354,14 @@ def test_action_status(self, mock_solo):
         dropdown_button = action_element.locator(".actions__status-selector")
         dropdown_content = action_element.locator(".dropdown__content")
 
-        # check state
-        expect(dropdown_button).to_contain_text(
-            str(StatusChoices.labels[StatusChoices.open])
-        )
+        approval_label = str(StatusChoices.approval.label)
+        closed_label = str(StatusChoices.closed.label)
```

Comment:
Does this need to be cast to a string?

Reply:
The labels are lazy strings, which will cause the test to fail. I'll add a comment to make this explicit.